platform/macos/SCsub

```diff
@@ -129,6 +131,12 @@ files = [
 
 prog = env.add_program("#bin/godot", files)
 
+env.Command(
+    "#misc/dist/macos_tools.app/Contents/Resources/Godot.icns",
+    "#platform/macos/icons/Godot_" + env.version_info["status"] + ".icns",
+    Copy("$TARGET", "$SOURCE"),
+)
```

Comment:
Probably should be copied in the `generate_bundle` instead (directly to target bundle), icon is not used outside of bundle.